R/mod_sentiment_distribution_plot.R

```diff
@@ -43,9 +43,14 @@ mod_sentiment_distribution_plot_server <- function(input, output, session, comme
   output$sentiment_plot <- renderPlot({
     req(comments_promise())
     comments_promise() %...>% {
-        ggplot(., aes(x = sentiment)) + 
-          geom_density(alpha = 0.5, fill = "lightblue")
-      }
+        comments <- .
+        validate(need(!is.null(comments), "Unfortunately there are no comments for this story."))
+        comments %>% pull(text)
+      } %...>% {
+          ggplot(., aes(x = sentiment)) + 
+            geom_density(alpha = 0.5, fill = "lightblue") +
+            xlim(-10, 10)
```

Comment:
Wouldn't it be better to keep those numbers as a variable in `globals.R` instead of using magic numbers :P ?